docs/application/dotnet/getting-started/runtime.md

```diff
@@ -0,0 +1,24 @@
+<a id="runtime"></a>
+# Managed Runtime Advantages
+
+A managed runtime offers the following advantages to your application development:
+
+- **Faster development**
+
+  Application development is accelerated since the managed runtime handles many functions that otherwise have to be coded into the application. The Tizen Native API and Tizen Web API already provide this benefit.
+
+- **Safer code**
+
+  Managed runtimes can handle, for example, bound checking, type safety, garbage collection, memory protection services, and objects being invoked directly. The Tizen Web API already provides some of these benefits, but not type safety, as JavaScript is not a strongly typed language.
```

Comment:
Managed runtime can handle bound checking, type safety, garbage collection, memory protection services, and objects that are invoked directly. The Tizen Web API already provides some of the benefits mentioned, but does not provide type safety, as JavaScript is not a strongly typed language. 

Is it bounds checking or bound checking?

Reply:
done. It is `bound checking`.